diff --git a/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js b/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js
index a44850d..e3ddd2c 100644
--- a/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js
+++ b/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js
@@ -1672,12 +1672,21 @@ if ( (process.env.NODE_ENV !== 'production')) {
 }
 const map = new Map();
 function registerHMR(instance) {
-    map.get(instance.type.__hmrId).instances.add(instance);
+  let record = map.get(instance.type.__hmrId)
+  if (!record) {
+    record = {
+      comp: instance.type,
+      instances: new Set(),
+    }
+    map.set(id, record)
+  }
+  record.instances.add(instance);
 }
 function unregisterHMR(instance) {
     map.get(instance.type.__hmrId).instances.delete(instance);
 }
 function createRecord(id, comp) {
+  debugger
     if (map.has(id)) {
         return false;
     }
